shoulder/config.py

```diff
@@ -211,6 +211,11 @@ def validate(self):
 c.description = "Path the shoulder test directory"
 config.add_configuration(c)
 
+_xml_register_dir = os.path.abspath(os.path.join(_project_dir, "../ARMv85A-SysReg-00bet9/SysReg_v85A_xml-00bet9"))
```

Comment:
I added a configuration to the XML register directory, with a default value of the newest spec located one directory above where shoulder is located. You can override it by setting ```shoulder.config.xml_register_dir = "/you/custom/path"```

Reply:
Also a great quality of life change. Should we specify it as the SysReg XML directory in case we want to parse the ISAs too and we can differentiate between the two?

Reply:
Agreed, I was thinking we could use this config (xml_register_dir) to point to the registers, and add one later for the a64 and a32 instruction directories.

Would the name  ```xml_sysreg_dir``` be better for this config?  To your point, xml_register_dir is slightly inaccurate, I'd be happy to change it.

Reply:
I think this is fine, it should be intuitive enough